src/graph_gui.cpp

```diff
@@ -298,9 +311,96 @@ struct BaseGraphWindow : Window {
 		return max_width;
 	}
 
+	/**
+	 * Draw one graph plot and all de data points.
+	 * It also determines the minimal distance between the mouse click (this->click), and the current data point
+	 * @param id the ID of the plot about to be drawn
+	 * @param r bounds of the graph
+	 * @param x_sep separation between data points on the x axis
+	 * @param x_axis_offset distance between the top of the graph and the x axis
+	 * @param interval_size absolute difference between the highest and the lowest data point
+	 * @param draw_selected controls the drawing of the highlighted selection
+	 * @return the minimal distance between all the data points in the current plot and the click point
+	 */
+	int DrawLineAndDots(int id, Rect r, int x_sep, int x_axis_offset, int interval_size, int draw_selected) const {
+		int x, y;
+		uint linewidth = _settings_client.gui.graph_line_thickness;
+		uint pointoffs1 = (linewidth + 1) / 2;
+		uint pointoffs2 = linewidth + 1 - pointoffs1;
+		/* The minimal (Manhattan) distance between the last click coordinate and all of the data points in the current plot */
```

Comment:
This seems to describe the following line, which is definitely not what the comment says. Maybe `Calculate the minimal (Manhattan) distance ...` would be more appropriate?

Reply:
Ah, in that case I'd remove the comment entirely. It's just a variable declaration that isn't important until it gets set later, so the comment only draws unwanted attention. 🙂